ci_framework/roles/edpm_deploy/README.md

```diff
@@ -14,7 +14,7 @@ None
 * `cifmw_edpm_deploy_retries`: (Integer) Number of retries for edpm deploy status. Defaults to `100`.
 * `cifmw_edpm_deploy_run_validation`: (Boolean) Run validation on EDPM node. Defaults to `False`.
 * `cifmw_edpm_deploy_dryrun`: (Boolean) Do a dry run on make edpm_deploy command. Defaults to `False`.
-* `cifmw_edpm_deploy_nova_pod_name`: (String) Name of the nova deployment pod started by an EDPM deploy. Defaults to `nova-edpm-compute-0-deploy-nova`.
```

Comment:
Sanity check: this was recently added. Are you sure you want to remove that? If so, shouldn't it be part of its own PR?

Reply:
I saw it, and yes, I'm sure. Ok, I'll open a separate PR for that part.